### PR TITLE
{tools}[gompi/2020a] mpifileutils v0.10.1

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/18e63d984d8bcb275715416026197e25 for a full test report.

File: easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.10.1-gompi-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+source_urls = ['https://github.com/hpc/%(name)s/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['mpifileutils-%(version)s_fix_acl_xattr_problem.patch']
+checksums = [
+    '4c8409ef4140f6f557d0e93f0c1267baf5d893c203b29fb7a33d9bc3c5a5d25c',  # v0.10.1.tar.gz
+    # mpifileutils-0.10.1_fix_acl_xattr_problem.patch
+    'e540ceaeb99758f4776252069d56b703411824ba83f48b474fc18785868151a2',
+]
+
+checksums = []
```
Comment (boegel) [inline easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.10.1-gompi-2020a.eb:23]: @akesandgren This looks like a mistake? :)

**Review by boegel — DISMISSED** [review]

File: easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.10.1-gompi-2020a.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.10.1-gompi-2020a.eb:23]: I wonder what new-pr did, there is a checksum in my file

Comment (akesandgren) [reply easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.10.1-gompi-2020a.eb:23]: Ahhh, --inject-checksums has a bug!!!

Comment (akesandgren) [reply easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.10.1-gompi-2020a.eb]: See https://github.com/easybuilders/easybuild-framework/issues/3459

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
node3404.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/0fad0e9b9d3364662efcb476d467ead3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2450.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/8cdad17bec7e745ea0bc65231fb00bc4 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**